services/convert/issue.go

```diff
@@ -238,3 +239,24 @@ func ToAPIMilestone(m *issues_model.Milestone) *api.Milestone {
 	}
 	return apiMilestone
 }
+
+// ToLabelTemplate converts Label to API format
+func ToLabelTemplate(label *label.Label) *api.LabelTemplate {
+	result := &api.LabelTemplate{
+		Name:        label.Name,
+		Exclusive:   label.Exclusive,
+		Color:       strings.TrimLeft(label.Color, "#"),
```

Comment:
I guess it's better to pass the value as is.

In case there will be some named colors like `red` in the future.

Reply:
That' the same as the API for Repo Label does. And when we change it in the Future, it will break. The API returns currently Colors only as Hex, so Programs using this API will assume they are always Hex, no matter if we put a # at the start or not.